webapp/src/modules/wallet/sagas.js

```diff
@@ -73,22 +89,28 @@ function* handleConnectWalletRequest(action = {}) {
 
     const manaTokenContract = eth.getContract('MANAToken')
     const landRegistryContract = eth.getContract('LANDRegistry')
+    const rcnTokenContract = eth.getContract('RCNToken')
     const marketplaceAddress = getMarketplaceAddress()
+    const mortgageCreatorAddress = getMortgageCreatorAddress()
+    const mortgageManagerAddress = getMortgageManagerAddress()
 
     const [
       network,
       balance,
       ethBalance,
       approvedBalance,
-      isLandAuthorized
+      isLandAuthorized,
+      isMortgageApprovedForMana,
+      isMortgageApprovedForRCN
```

Comment:
maybe these two should be called `mortgageManaAllowance` and `mortgageRCNAllowance` ? so then 
      isMortgageApprovedForMana: isMortgageApprovedForMana > 0,
      isMortgageApprovedForRCN: isMortgageApprovedForRCN > 0

Reply:
Yes, it is better

Reply:
☝️